app/components/CarbonTonsRetiredCard/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Text } from "@klimadao/lib/components";
+import ForestOutlinedIcon from "@mui/icons-material/ForestOutlined";
+import * as styles from "./styles";
+import { FC } from "react";
+import { Trans } from "@lingui/macro";
+
+interface Props {
+  isConnected?: boolean;
+}
+
+export const CarbonTonsRetiredCard: FC<Props> = () => {
```

Comment:
nitpick, always use metric Tonnes (1,000 kg)
```suggestion
export const CarbonTonnesRetiredCard: FC<Props> = () => {
```